### PR TITLE
Implement local notifications and allow user to schedule them daily or weekly

### LINKED ISSUE #6: Implement local notifications
One of the main things for the app is the nagging to the user asking "What's your beautiful thing today?". At first I thought we needed to do that on the server, but it turns out we can do this on the client side.

Here's a cordova plugin that can be used, there might be better plugins, that's up to you to decide:

https://github.com/katzer/cordova-plugin-local-notifications

The app should periodically poll the preferences to see whether the nagging configuration changed (weekly or daily), and based on that it should setup local notifications to happen at 10pm (local time for the device) either every day or every Sunday.

The notification should show the app logo (non existent for now, but use a placeholder logo). And as mentioned before it should read: What's your beautiful thing today?

When the user taps on the notification it should take them directly to the beautiful thing creation pane, ready to write.

### PR DESCRIPTION
**Pull request by fran-costa**
Close #6

### CODE REVIEW

File: react-app/src/notifications/index.js [hunk 1]
```diff
@@ -1,4 +1,60 @@
 const DAILY = 1;
 const WEEKLY = 2;
 
-export { DAILY, WEEKLY }
+const SUNDAY_WEEKDAY = 7;
+
+function _showAddScreen() { /** TODO */ }
+
+function setNotifications(shcedule = DAILY) {
```
Comment (leanbalma) [inline react-app/src/notifications/index.js]: Typo: "shcedule".

File: react-app/src/notifications/index.js [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+  if (!window.cordova) return;
+
+  const notification = {
+    id: shcedule,
+    title: 'BeautifulThingsApp',
```
Comment (leanbalma) [inline react-app/src/notifications/index.js]: We should display the name of the app here, right? Why the "App" at the end and not just "BeautifulThings"?

File: react-app/src/notifications/index.js (hunk 1 above)
Comment (fran-costa) [reply react-app/src/notifications/index.js]: Done.

File: react-app/src/notifications/index.js (hunk 2 above)
Comment (fran-costa) [reply react-app/src/notifications/index.js]: When I started working, I thought that the full name of the app was BeautifulThingsApp, and used this title until now.

I've just changed it to "Beautiful Things".

**Pull request merged**